aten/src/ATen/native/Activation.cpp

```diff
@@ -606,6 +606,7 @@ Tensor& rrelu_with_noise_out_cpu(const Tensor& self,
     bool training,
     c10::optional<Generator> generator,
     Tensor& output) {
+  TORCH_CHECK(self.sizes() == noise.sizes(), "noise tensor shape must match self tensor shape. Got self.shape = ", self.sizes(), " noise.shape = ", noise.sizes())
```

Comment:
Should this be `sym_sizes()` to prevent issues with symbolic tensors?

Reply:
Fixed, thanks!